### PR TITLE
Rework disputes descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks the disputes descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Common/CommonTransaction.yaml [hunk 1]
```diff
@@ -138,67 +152,75 @@ properties:
...
   createdTime:
-    description: Transaction created time.
+    description: Date and time in which the transaction was created.
     x-label: Created On
     x-sortable: true
     allOf:
       - $ref: ../ServerTimestamp.yaml
   updatedTime:
-    x-label: Last update time
+    x-label: Date and time in which the transaction was last updated.
```
Comment (Boydbueno) [inline openapi/components/schemas/Common/CommonTransaction.yaml:223]:
```suggestion
    x-label: Date and time at which the transaction was last updated.
```

File: openapi/components/schemas/Common/CommonTransaction.yaml [hunk 2]
```diff
@@ -138,67 +152,75 @@ properties:
...
     type: boolean
     readOnly: true
   3ds:
+    description: 3D Secure object.
     allOf:
       - $ref: ../Transactions/ThreeDSecureResult.yaml
   redirectUrl:
-    description: The URL to redirect the end-user when an offsite transaction is completed. Defaults to the website's configured URL.
+    description: |-
+      URL where the end-user is redirected when an offsite transaction
```
Comment (Boydbueno) [inline openapi/components/schemas/Common/CommonTransaction.yaml:164]: I think this is a bit clearer
```suggestion
      URL where the end-user is redirected to when an offsite transaction
```

File: openapi/components/schemas/Common/CommonTransaction.yaml [hunk 3]
```diff
@@ -97,39 +105,45 @@ properties:
...
   invoiceIds:
-    description: The invoice IDs related to transaction.
+    description: Related invoice IDs.
     readOnly: true
     type: array
     items:
       $ref: ../ResourceId.yaml
   subscriptionIds:
-    description: The orders IDs related to transaction's invoice(s).
+    description: Orders IDs of invoices that are related to the transaction.
```
Comment (Boydbueno) [inline openapi/components/schemas/Common/CommonTransaction.yaml:125]:
```suggestion
    description: Subscription IDs of invoices that are related to the transaction.
```

File: openapi/components/schemas/GatewayAccount.yaml [hunk 4]
```diff
@@ -229,120 +229,139 @@ properties:
...
+      `reconciliationWindowTtl` time, the transaction is marked as
+      `abandoned`.
     type: boolean
     default: false
   reconciliationWindowTtl:
-    description: >-
-      The time window (in seconds) allotted for a reconciliation to occur.
-      If it is not reconciled in that time, then the transaction is marked as `abandoned`.
+    description: |-
+      Allotted time, in seconds, in which a must reconciliation to occur
```
Comment (Boydbueno) [inline openapi/components/schemas/GatewayAccount.yaml:270]:
```suggestion
      Allotted time, in seconds, in which a reconciliation must occur
```

File: openapi/components/schemas/Transactions/Transaction.yaml [hunk 5]
```diff
@@ -199,36 +222,45 @@ allOf:
...
-        description: True if the transaction was initiated by the merchant.
+        description: |-
+          Specifies whether the transaction was initiated by the merchant.
         type: boolean
       hadDiscrepancy:
-        description: >-
-          True if the transaction has been updated due to a discrepancy with its.
-          source of truth.
+        description: |-
+          Specifies whether the transaction was been updated due to a
```
Comment (Boydbueno) [inline openapi/components/schemas/Transactions/Transaction.yaml:241]:
```suggestion
          Specifies whether the transaction was updated due to a
```

File: openapi/components/schemas/Common/CommonTransaction.yaml [hunk 6]
```diff
@@ -138,67 +152,75 @@ properties:
...
+      gateway is selected for the transaction.
     x-label: Gateway account
     x-basic: true
     allOf:
       - $ref: ../Gateways/GatewayName.yaml
   customFields:
     $ref: ../ResourceCustomFields.yaml
   processedTime:
-    description: Transaction processed time.
+    description: Date and time in which the transaction was processed.
```
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/Common/CommonTransaction.yaml:211]: It be should `at`! I was consistently incorrect here 😅

File: openapi/components/schemas/Common/CommonTransaction.yaml (hunk 2 above)
Comment (noon-dawg) [reply openapi/components/schemas/Common/CommonTransaction.yaml:164]: Looks good

File: openapi/components/schemas/GatewayAccount.yaml [hunk 7]
```diff
@@ -229,120 +229,139 @@ properties:
...
-    description: Gateway Account token.
+    description: Gateway account token.
     type: string
     readOnly: true
     example: TwiX3f92k4AiBE27BzTbQ38hHjicBz_w
   sticky:
-    description: Customer's payment instrument will "stick" to the gateway account for future transactions when enabled.
+    description: |-
+      Specifies whether all future payments by the customer's payment
+      instrument will be processed by the same gateway account.
```
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/GatewayAccount.yaml:317]: Excellent input!

File: openapi/components/schemas/Common/CommonBankAccount.yaml [hunk 8]
```diff
@@ -61,7 +62,7 @@ properties:
   useAsBackup:
     $ref: ../UseAsBackup.yaml
   createdTime:
-    description: Bank account created time.
+    description: Date and time at which the bank account was created .
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonBankAccount.yaml:65]:
```suggestion
    description: Date and time at which the bank account was created.
```

**Review by ashkarpetin — APPROVED** [review]

**Review by Boydbueno — APPROVED** [review]

**Pull request merged**